doc/man1/ciphers.pod

```diff
@@ -346,7 +346,7 @@ Cipher suites using SHA1.
 
 =item B<SHA256>, B<SHA384>
 
-Ciphersuites using SHA256 or SHA384.
+Cipher suites using SHA256 or SHA384.
```

Comment:
We seem to be inconsistent about this. I get 438 matches for "ciphersuite" in the codebase and 128 for "cipher suite". Personally I prefer the former.